Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
```

Comment:
### [rspec-mocks](https://github.com/rspec/rspec-mocks)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.11.3 → 3.11.1

[_[change-log](https://github.com/rspec/rspec-mocks/blob/v3.11.1/Changelog.md), [source-code](https://github.com/rspec/rspec-mocks), [gem-diff](https://my.diffend.io/gems/rspec-mocks/2.11.3/3.11.1)_]





<details>
<summary>Commits</summary>

A change of **2040** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-mocks/compare/v2.11.3...v3.11.1).

These are the first 10 commits:
- (4847ef0) [Changelog for #1167.](https://github.com/rspec/rspec-mocks/commit/4847ef067b60e3725a9e4819b9d0ee0f5c5262eb)
- (d8e2d47) [Update README.md (#1169)](https://github.com/rspec/rspec-mocks/commit/d8e2d4778eeb4117bea8a1bc9135a44720e65505)
- (0820af7) [Updated travis build scripts (from rspec-dev) (#1172)](https://github.com/rspec/rspec-mocks/commit/0820af734f1a8926b0fee4fa4dd3e00ff70b4990)
- (2f64278) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-mocks/commit/2f642786a93129e3408d4dc4843fb38409b90f88)
- (8b010b7) [Merge pull request #1174 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-mocks/commit/8b010b7a774570ceca86fb7bf6b0cc9da9a28eb0)
- (fa91024) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-mocks/commit/fa910249324822f2ebaac9ab9b7c10a8f6b7b701)
- (297dd60) [Mark a pending example on JRuby 9000 fixed](https://github.com/rspec/rspec-mocks/commit/297dd60fcee36d7eb87b3b03e8dad2e8736a344d)
- (78d04f0) [Merge pull request #1180 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-mocks/commit/78d04f05d45ef920afb0a51f118b119898c76ab3)
- (7b8bb80) [Updates changelog for v3.7.0 [ci skip]](https://github.com/rspec/rspec-mocks/commit/7b8bb8010cee0fc1a15982e90f3793105cff1f07)
- (df6ce2c) [Releases 3.7.0](https://github.com/rspec/rspec-mocks/commit/df6ce2cc4a4bb2f5d62a2490fe4f2c290c22590c)


</details>